script.js

```diff
@@ -1,4 +1,709 @@
+          resultLog =
+            resultLog +
+            `Player ${
+              playerCounter + 1
+            }'s numbers in descending order: ${sortDes(
+              competingArrays[playerCounter]
+            )}. (Total: ${competingTotals[playerCounter]}) <br>`;
+          playerCounter += 1;
+        }
+        playerNo += 1;
+        // Output Result message
+        return (
+          `You rolled these numbers: ${unsortedDiceRollArray}. <br> <br> This is your best combined number from those digits above: ${bestNumber}. <br><br><br> 🏆 THE WINNER WHO PROGESSED TO THE NEXT ROUND IS: PLAYER ${winningPlayer}! 🏆 <br> <br> You are currently in the ${resultMode} game mode. <br> <br>` +
+          resultLog +
+          `<br><br><br> To continue, please get the next player to roll the dice by clicking the submit button. <br><br><br> ================= <br><br> Tip: <br><br> Enter "reversed" for reversed mode (lowest number wins). <br> Enter "normal" for normal mode (highest number wins).`
+        );
+      }
+      // If this is the final player's turn, output the current player's dice roll outcome, along with win/lose results and stats
+      if (playerNo == numberOfPlayers - 1) {
+        // Find winner based on running sum thus far
+        winningPlayer = findWinner(competingTotals, resultMode);
+        console.log("competingTotals", competingTotals);
+        console.log("winningPlayer", winningPlayer);
+        // Print the competing players' numbers and total sum (for message output use)
+        playerCounter = 0;
+        resultLog = ``;
+        while (playerCounter < numberOfPlayers) {
+          resultLog =
+            resultLog +
+            `Player ${
+              playerCounter + 1
+            }'s numbers in descending order: ${sortDes(
+              competingArrays[playerCounter]
+            )}. (Total: ${competingTotals[playerCounter]}) <br>`;
+          playerCounter += 1;
+        }
+        // Reset variables in preparation for fresh round after outcome is generated
+        playerNo = 0;
+        numberOfDice = UNASSIGNED;
+
+        // Output Result message
+        return (
+          `You rolled these numbers: ${unsortedDiceRollArray}. <br> <br> This is your best combined number from those digits above: ${bestNumber}. <br><br><br> 🏆 THE FINAL WINNER OF THE ROUND IS: PLAYER ${winningPlayer}! 🏆 <br> <br> You are currently in the ${resultMode} game mode. <br> <br>` +
+          resultLog +
+          `<br><br><br> To continue, please get Player 1 to enter the number of dice you wish to roll for the next round. <br><br><br> ================= <br><br> Tip: <br><br> Enter "reversed" for reversed mode (lowest number wins). <br> Enter "normal" for normal mode (highest number wins).`
+        );
+      }
+      // Reset secondary arrays. Do not reset core array "competingNumbers"
+      competingArrays = [];
+      competingTotals = [];
+      indivArray = [];
+      indivTotal = [];
```

Comment:
A restart function like mentioned at the top will be good to call here too! 
Overall, very proud of the work you've done Eunice!!